libs/cardano-ledger-core/src/Cardano/Ledger/Address.hs

```diff
@@ -193,12 +211,21 @@ addrPtrNormalize = \case
   addr -> addr
 
 -- | An account based address for rewards
-data RewardAcnt c = RewardAcnt
+data RewardAccount c = RewardAccount
   { getRwdNetwork :: !Network
   , getRwdCred :: !(Credential 'Staking c)
```

Comment:
While you are at it, let's rename these two as well. The convention adopted couple years ago (after this type creation) is to prefix fields with abbreviation of the type, thus the `ra` prefix:
```suggestion
  { raNetwork :: !Network
  , raCredential :: !(Credential 'Staking c)
```